warehouse/templates/base.html

```diff
@@ -113,9 +113,13 @@
     <meta property="og:url" content="{% if request.matched_route %}{{ request.current_route_url() }}{% else %}{{ request.url }}{% endif %}">
     <meta property="og:site_name" content="PyPI">
     <meta property="og:type" content="website">
+    {% if show_share_image %}
```

Comment:
...and have this become a [Jinja filter](https://github.com/pypi/warehouse/blob/9a270a2d01cba250795daf1e05133fbbafeeb7b0/warehouse/filters.py) instead
```suggestion
    {% if user_agent | show_share_image %}
```
That way, we can just write additional filters if we need to in the future, rather than adding to the return value for `release_detail`.